astro.config.mjs

```diff
@@ -12,4 +12,7 @@ export default defineConfig({
   integrations: [vue(), react()],
   output: "server",
   adapter: vercel(),
-});
\ No newline at end of file
+  experimental: {
+    responsiveImages: true,
+  },
+});
```

Comment:
Legg merke til denne. Den gjør at vi kan sette layout="full-width" for å få på en del css klasser og src set på bilder